.github/workflows/skms.yaml

```diff
@@ -0,0 +1,130 @@
+name: Create CMR in SKMS
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
+
+jobs:
+  create-cmr:
+    # Only run this workflow on pull requests that have merged and not manually closed by user
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # Runs a single command using the runners shell for shell validation
+      - name: Validate Shell
+        run: echo Starting CMR Action...
+
+      - name: Check Dependencies
+        run: |
+          if ! type "jq" >/dev/null; then
+            echo "jq is required but not installed"
+            exit 1
+          fi
+
+          if ! type "curl" >/dev/null; then
+            echo "curl is required but not installed"
+            exit 1
+          fi
+
+          echo "Dependencies check was successful"
+
+      - name: Set Maintenance Time Windows for CMR
+        run: |
+          echo "start_time=$(date -d "+60 minutes" '+%Y-%m-%d %H:%M')" >> $GITHUB_ENV
+          echo "end_time=$(date -d "+90 minutes" '+%Y-%m-%d %H:%M')" >> $GITHUB_ENV
+
+      - name: Set Release Summary for CMR
+        run: |
+          function sanitizeStr() {
+            local str=$1
+
+            if [ -z "$str" ]; then
+              echo "First parameter missing, must be a valid string"
+              return 1
+            fi
+
+            str="${str//\"/""}"
+            str="${str//\'/""}"
+            str="${str//(/"["}"
+            str="${str//)/"]"}"
+            str="${str//$'\r'/"\r"}"
+            str="${str//$'\n'/"\n"}"
+            str="${str//$'\t'/"\t"}"
+            str="${str//\\//}"
+            str="${str////"\/"}"
+
+            echo "$str"
+          }
+
+          release_title=$(sanitizeStr "${{ github.event.pull_request.title }}")
+          release_details=$(sanitizeStr "${{ github.event.pull_request.body }}")
+          release=Release_Title--"${release_title}"--Release_Details--"${release_details}"--Pull_Request_Number--"${{ github.event.pull_request.number }}"--Pull_Request_Created_At--"${{ github.event.pull_request.created_at }}"--Pull_Request_Merged_At--"${{ github.event.pull_request.merged_at }}"
+
+          echo "release_summary<<RS_EOF" >> $GITHUB_ENV
+          echo "$release" >> $GITHUB_ENV
+          echo "RS_EOF" >> $GITHUB_ENV
+
+      - name: Create CMR in SKMS
+        run: |
+          DEFAULT_SKMS_URL='api.skms.adobe.com'
+
+          function skms_request() {
+            local username=$1
+            local passkey=$2
+            local object=$3
+            local method=$4
+            local method_params=$5
+            local url=$6
+
+            if [ -z "$username" ]; then
+              echo "First parameter missing, must be SKMS username"
+              return 1
+            fi
+
+            if [ -z "$passkey" ]; then
```

Comment:
Just to be sure, passkey is not a secret - correct? Since we log this and this is public.

Reply:
@mokimo, excellent question, it is a secret that is coming in from a GitHub repository secret variable so it is masked as '*' in the logs as shown below in the GitHub Actions Workflow printout:

![Screenshot 2023-12-05 at 9 49 29 AM](https://github.com/adobecom/milo/assets/56042473/c4926d0d-1651-4354-8ed4-4eb6c8e829c8)

and

![Screenshot 2023-12-05 at 9 49 39 AM](https://github.com/adobecom/milo/assets/56042473/7e377f25-bc31-4005-beff-262511df07e3)

So we should be good.